### ISSUE TITLE
Could not build on Windows 10 through Cmake with Mingw files: could not find pyqt even though the version is satisfied

### ISSUE THREAD

**Issue by Seekerzero** [Ultimaker/cura-build#241]
Found Python3: C:/Users/Seeker Wong/AppData/Local/Programs/Python/Python35/python.exe (found version "3.5.4") found components: Interpreter 
CMake Warning (dev) at D:/autogcode/Cura/Cura-Building-Environments/libArcus-mingw/install_dir/lib/cmake/Arcus/ArcusConfig.cmake:30 (set):
  implicitly converting '' to 'STRING' type.
Call Stack (most recent call first):
  projects/CuraEngine.cmake:1 (find_package)
  CMakeLists.txt:106 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

Found Python3: C:/Users/Seeker Wong/AppData/Local/Programs/Python/Python35/python.exe (found suitable version "3.5.4", minimum required is "3.4") found components: Interpreter 
CMake Error at D:/Program Files/CMake/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:160 (set):
  Syntax error in cmake code at

    D:/Program Files/CMake/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:160

  when parsing string

    Could NOT find PyQt: Found unsuitable version "Wong\AppData\Local\Programs\Python\Python35\lib\site-packages\PyQt5\QtCore.pyd 5.15.0", but required is at least "5.10" (found C:/Users)

  Invalid character escape '\A'.
Call Stack (most recent call first):
  D:/Program Files/CMake/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:456 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindPyQt.cmake:8 (find_package_handle_standard_args)
  projects/Uranium.cmake:7 (find_package)
  CMakeLists.txt:106 (include)

Configuring incomplete, errors occurred!
See also "D:/autogcode/Cura/cura-build/build/CMakeFiles/CMakeOutput.log".

**Comment by Seekerzero** [comment]
Also, another question: When I add some new features in the Cura, do I need to build it to proceed the changes, or just need to run the cura_app.py?

**Comment by Seekerzero** [comment]
to be mentioned, I first tried PyQt 5.10 and it still show the version is incorrect.

**Comment by nallath** [comment]
You only need to run cura_app.py. Building is only to make a distributed file (unless you make changes to the engine, in that case you do need to build)

**Comment by Seekerzero** [comment]
Thank you so much!
btw, how to print out or logging information in Cura? I found Print() Function is not working. Is there anything block that?

**Comment by Seekerzero** [comment]
I found no info print out from the terminal. that is the reason that I ask if I only need to run cura_app.py to get changes .

**Comment by Seekerzero** [comment]
Another example is that I change a text description for a button in RotateTool.qml (inside Tools folder), and it didn't change when I re-ran the cura_app.py.

**Comment by nallath** [comment]
You can use either Logger.log("d", "whatever") or print() both work for me.

If it didn't work, you probably haven't set it up correctly (so it's using different code)

**Comment by Seekerzero** [comment]
That's weired.

**Comment by Seekerzero** [comment]
I found the reason! well I thought the plugins in Cura are actually running. but they are not. those are actually running under UM/plugins.
But what's the reason that is required to copy plugins and resources folders from UM to Cura when we set up the environment?
The following part is from https://github.com/Ultimaker/Cura/wiki/Running-Cura-from-Source-on-Windows

Uranium

Uranium, or UM, is a required library, but does not need to be compiled.

    Clone the Uranium repository:

    $ git clone https://github.com/Ultimaker/Uranium.git C:/dev/Uranium

    Add the Uranium directory to your PYTHONPATH environment variable. It can be done with the following command, even if PYTHONPATH is not set up:

    set PYTHONPATH=%PYTHONPATH%;C:\dev\Uranium

**If you decide to add UM to the main Python install, the contents of the plugins and resources directories will need to be copied to the C:/dev/Cura/plugins and C:/dev/Cura/resources directories respectively when the Cura repository is cloned.**
